Runners/ConsoleApp/.filenesting.json

```diff
@@ -0,0 +1,3 @@
+{
+    "help":"https://go.microsoft.com/fwlink/?linkid=866610"
```

Comment:
What is this file?

Reply:
Unnecessary.
Probably a result of the attempts to work with the resources